packages/react-scripts/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # `backpack-react-scripts` Change Log
 
+## 10.1.0
+
+- Disable `loadable` in CSR build
```

Comment:
Even though it is a big change to CSR build, but no any breaking is here for BRS, so a `minor` version is OK.